static/css/add.scss

```diff
@@ -0,0 +1,266 @@
+.Page {
+  text-align: center;
+}
+
+.Form {
+  display: block;
+  // Page title
+  &__title {
+    font-family: $title;
+    font-size: 2em;
+    font-weight: 400;
+    color: $links;
+    display: block;
+    text-align: center;
+    margin-bottom: 0.25em;
+  }
+
+  // Boundary box around form
+  &__box {
+    display: block;
+    width: 32.5em;
+    border: 0.05em solid $main;
+    border-radius: 0.5em;
+    padding: 1em 1.5em;
+    margin-right: auto;
+    margin-left: auto;
+    margin-bottom: 1em;
+    box-shadow: 0.1em 0.1em 0.25em $secondary;
+  }
+
+  // Row for duration section
+  &__row {
+    display: block;
+
+    h2 {
+      display: block;
+      text-align: left;
+      font-family: $title;
+      font-size: 1em;
+      font-weight: 800;
+      color: $form-header;
+      text-transform: uppercase;
+      padding-left: 0.1em;
+      margin-bottom: 0.5em;
+    }
+
+    &__buttons {
+      display: flex;
+
+      input {
+        display: none;
+      }
+
+      label {
+        align-content: flex-start;
+        font-family: $title;
+        font-size: 1em;
+        font-weight: 800;
+        color: white;
+        text-transform: uppercase;
+        padding: 0.5em 0.5em;
+        margin: 0em 0.5em 0.5em 0em;
+        border: none;
+        border-radius: 0.5em;
+        background-color: $links-light;
+        transition-duration: 0.3s;
+
+        &:hover {
+          background-color: $links;
+        }
+
+        &:focus {
+          background-color: $links;
+        }
+      }
+    }
+  }
+
+  // Row for Date-Time sections
+  &__date {
+    display: flex;
+
+    label {
+      display: block;
+      text-align: left;
+      font-family: $title;
+      font-size: 1em;
+      font-weight: 800;
+      color: $form-header;
+      text-transform: uppercase;
+      padding-left: 0.1em;
+      margin-bottom: 0.5em;
+    }
+
+    input {
+
+    }
+  }
+
+  // Styling dropdown menu
+  &__category {
+    overflow: hidden;
+
+    select {
+      display: block;
+      font-family: $title;
+      font-size: 1em;
+      font-weight: 800;
+      text-transform: uppercase;
+      color: $secondary;
+      background-color: white;
+      border: 0.05em solid $main;
+      border-radius: 0.5em;
+      padding: 0.5em 0.5em;
+      margin-bottom: 0.5em;
+      transition: box-shadow 0.3s;
+
+      // Changing the dropdown arrow to a chevron
+    	background: url("http://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/br_down.png") right center no-repeat;
+    	appearance: none;
```

Comment:
Should probably find a different method of changing the appearance.

You should probably do:
```scss
select {
    appearance: none; // This gets rid of default styling

    &::after {
        content: url("data:image/svg+xml;charset=UTF-8,<svg>...</svg>"); // replace the <svg> with the svg of the down arrow
        position: absolute;
        right: 0;
        height: 100%;
        top: 0;
    }
}
```
idk something like that. Not sure if it'll work. The issue is that when it renders, there is no padding on the right side.

![2020-07-09_21-01_screenshot](https://user-images.githubusercontent.com/7565220/87115184-5f80a180-c227-11ea-85a7-0bf6ae874ae6.png)